etc/adminhtml/system.xml

```diff
@@ -5,12 +5,25 @@
         <section id="system" translate="label" type="text" sortOrder="900" showInDefault="1">
             <tab>advanced</tab>
             <resource>Aligent_AsyncEvents::manage</resource>
-            <group id="async_events" translate="label" type="text" sortOrder="1000" showInDefault="1">
+            <group id="async_events" translate="label" type="text" sortOrder="1000" showInDefault="1" showInStore="1" showInWebsite="1">
                 <label>Async Events</label>
-                <field id="max_deaths" translate="label" type="text" sortOrder="10" showInDefault="1">
+                <field id="max_deaths" translate="label" type="text" sortOrder="10" showInDefault="1" showInStore="1" showInWebsite="1">
                     <label>Maximum Deaths</label>
                     <validate>integer validate-not-negative-number</validate>
                 </field>
+                <field id="subscriber_log_cleanup_cron" translate="label" type="select" sortOrder="20" showInDefault="1" showInStore="1" showInWebsite="1">
```

Comment:
These configuration values can just be at the global level instead of website and store views.